docs/src/pages/HomePage/HomePage.js

```diff
@@ -180,10 +182,10 @@ const HomePage = (props, context) => {
         <div className={getClassName('bpkdocs-home-page__hero-inner')}>
           <BpkText
             textStyle="xl"
-            tagName="h1"
+            tagName="p"
```

Comment:
This is quite a substantial block of text, so shouldn't be a heading

Reply:
How does this style look now that it has changed to `p`? Might be worth to check it looks ok now too, otherwise we need to tweak the styles